src/DotNetStatus/Microsoft.Fx.Portability.ObjectModel/BreakingChangeAnalyzerStatusChartColors.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace Microsoft.Fx.Portability.ObjectModel
+{
+    public static class BreakingChangeAnalyzerStatusChartColors
+    {
+        public static string GetColor(BreakingChangeAnalyzerStatus breakingChangeAnalyzerStatus)
+        {
+            switch (breakingChangeAnalyzerStatus)
+            {
+                case BreakingChangeAnalyzerStatus.Available:
+                    return "#e2efda";
+                case BreakingChangeAnalyzerStatus.Investigating:
+                    return "#ff6a00";
+                case BreakingChangeAnalyzerStatus.NotPlanned:
+                    return "#f8cbad";
+                case BreakingChangeAnalyzerStatus.Planned:
+                    return "#fff2cc";
+                case BreakingChangeAnalyzerStatus.Unknown:
+                    return "#808080";
+                default:
+                    throw new ArgumentException(
+                        $"BreakingChangeAnalyzerStatusChartColors_GetColor: {breakingChangeAnalyzerStatus.ToString()} is not defined as a color!");
```

Comment:
Might be better to use this exception: https://msdn.microsoft.com/en-us/library/dsyf5bb2(v=vs.110).aspx.